src/parser/ast/debug_sym_factory.rs

```diff
@@ -0,0 +1,78 @@
+use codespan_reporting::files::{Files, SimpleFile};
```

Comment:
super good catch, I didn't know about this library and I thought we had to build this